pkg/membrane/membrane.go

```diff
@@ -141,15 +143,15 @@ func (s *Membrane) startChildProcess() error {
 	// TODO: This is a detached process
 	// so it will continue to run until even after the membrane dies
 
-	fmt.Println(fmt.Sprintf("Starting Child Process"))
+	log.Default().Println("Starting Child Process")
 	childProcess := exec.Command(s.childCommand[0], s.childCommand[1:]...)
 	childProcess.Stdout = os.Stdout
 	childProcess.Stderr = os.Stderr
 	applicationError := childProcess.Start()
 
 	// Actual panic here, we don't want to start if our userland code cannot successfully start
 	if applicationError != nil {
-		return fmt.Errorf("There was an error starting the child process: %v", applicationError)
+		return fmt.Errorf("there was an error starting the child process: %w", applicationError)
```

Comment:
Could this be a log.Default().Fatal if its an actual panic?

Reply:
no, it will return  an error to the caller. We shouldn't assume failure handling